lib/common_test/src/ct.erl

```diff
@@ -404,37 +404,37 @@ be specified as `Required`. Only the last element of the tuple can be a list of
 _Example 1._ Require the variable `myvar`:
 
 ```erlang
- ok = ct:require(myvar).
+ok = ct:require(myvar).
 ```
 
 In this case the configuration file must at least contain:
 
 ```text
- {myvar,Value}.
+{myvar,Value}.
 ```
 
 _Example 2._ Require key `myvar` with subkeys `sub1` and `sub2`:
 
 ```erlang
- ok = ct:require({myvar,[sub1,sub2]}).
+ok = ct:require({myvar,[sub1,sub2]}).
 ```
 
 In this case the configuration file must at least contain:
 
-```erlang
- {myvar,[{sub1,Value},{sub2,Value}]}.
+```text
+{myvar,[{sub1,Value},{sub2,Value}]}.
 ```
 
 _Example 3._ Require key `myvar` with subkey `sub1` with `subsub1`:
 
 ```erlang
- ok = ct:require({myvar,sub1,sub2}).
+ok = ct:require({myvar,sub1,sub2}).
 ```
 
 In this case the configuration file must at least contain:
 
-```erlang
- {myvar,[{sub1,[{sub2,Value}]}]}.
+```text
+{myvar,[{sub1,[{sub2,Value}]}]}.
```

Comment:
Shouldn't these two codeblocks stay as `erlang`?

Reply:
I'm not sure, it's a file not code. You think it should be erlang?

Reply:
So it should be erlang or text?

Reply:
terms should be `erlang`.

Reply:
Thanks for clarification, I will fix that here and look all over the code for other cases in common test.

Reply:
Fixed.